### PR TITLE
Alias bold to semibold (and change wherever possible right now)

### PR DESCRIPTION
**Pull request by phndiaye**
### What does this PR do?

Alias bold to semibold (and change wherever possible right now)

Related to: https://github.com/upfluence/backlog/issues/1807

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated tests
- [ ] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: app/styles/core-v2/_shims.less [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/* cyrillic-ext */
```
Comment (JulienVannier66) [inline app/styles/core-v2/_shims.less:1]: What is the goal of this file?

Comment (phndiaye) [reply app/styles/core-v2/_shims.less:1]: It forces Bold/700 font-weights to load the SemiBold font. This way we don't have to chase all Bold usages right now.

Comment (JulienVannier66) [reply app/styles/core-v2/_shims.less:1]: So it will be impossible to set bold in the soft after that

Comment (phndiaye) [reply app/styles/core-v2/_shims.less:1]: Yep !

**Review by Miexil — APPROVED** [review]

**Review by JulienVannier66 — APPROVED** [review]

**Pull request merged**